### PR TITLE
API Integrated Both side, Response Page added, payload added, SV Access Added

### PR DESCRIPTION
**Pull request by ShivankShuklaa**

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/SVAcknowledgement.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+const GetActionMessage = (props) => {
+    const { t } = useTranslation();
+    if (props.isLoading) {
+      return t("SV_APPLICATION_PENDING");
+    }
+    return props.isSuccess  ? t("SV_SUBMIT_SUCCESSFULL") : t("SV_APPLICATION_FAILED");
+  };
+
+const rowContainerStyle = {
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/SVAcknowledgement.js:34]: Here as well which component it provides style

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/index.js [hunk 2]
```diff
@@ -94,12 +93,10 @@ const SVCreate = ({ parentRoute }) => {
 
   const onSuccess = () => {
     clearParams();
-    queryClient.invalidateQueries("SV_CREATE");
+    queryClient.invalidateQueries("SV_CREATES");
+    sessionStorage.removeItem("beneficiary");
```
Comment (Keshav-NIUA) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/index.js:97]: Please add comments to every method as per the standard then I will merge

File: frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/utils/index.js [hunk 3]
```diff
@@ -235,6 +235,16 @@ const ewAccess = () => {
...
+  const userInfo = Digit.UserService.getUser();
+  const userRoles = userInfo?.info?.roles?.map((roleData) => roleData?.code);
+  const svRoles = ["SVCEMP", "TV CEMPLOYEE", "INSPECTION_OFFICER"];
+  const SV_ACCESS = userRoles?.filter((role) => svRoles?.includes(role));
+  return SV_ACCESS?.length > 0;
+};
+
+
+
 const chbAccess = () => {
```
[the remark this answers is not in the document]
Comment (ShivankShuklaa) [reply frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/utils/index.js:248]: commnents added

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/SVAcknowledgement.js (hunk 1 above)
Comment (ShivankShuklaa) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/SVAcknowledgement.js:34]: comment added this style object will provide style to the banner which shows the application status  and application id

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/index.js (hunk 2 above)
Comment (ShivankShuklaa) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/index.js:97]: Comments added

**Pull request merged**